src/nuclei/client/main.py

```diff
@@ -177,25 +231,33 @@ def get_application_version(self, app: str) -> str:
         ConnectionError:
             Application not available
         TypeError:
-            Wrong type for `app` argument
+            Wrong type for `app` or `version` argument
         ValueError:
-            Wrong value for `app` argument
+            Wrong value for `app` or `version` argument
```

Comment:
We are missing code to check this.

Reply:
This is a propagating error, its in self.get_url.